docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -115,12 +115,12 @@ using Plots
 oplt = plot(osol; title = "Reaction rate equation (ODE)")
 splt = plot(ssol; title = "Chemical Langevin equation (SDE)")
 jplt = plot(jsol; title = "Stochastic chemical kinetics (Jump)")
-plot(oplt, splt, jplt; lw = 2, size=(800,800), layout = (3,1)) 
-plot!(bottom_margin = 3Plots.Measures.mm) # hide
-nothing # hide
+plot(oplt, splt, jplt; lw = 2, size=(800,800), layout = (3,1)) # hide
```

Comment:
Now this is silly. If i put the # hide on the last line it does not work, and you see that line extra (including `# hide` at the end). No clue what to do about it, but I have asked about it so maybe the docs people can help with a fix.

Reply:
(I am happy to leave this one be until we can figure out what is actually going on here)